PocketKit/Sources/PocketKit/Tags/MailView.swift

```diff
@@ -0,0 +1,54 @@
+import AVFoundation
+import Foundation
+import MessageUI
+import SwiftUI
+import UIKit
```

Comment:
what's this `import UIKit` here for?

Reply:
because from what I saw, there is no swiftUI native mail view and I have to implement it with UIKit

Reply:
sure but I think `UIViewControllerRepresentable` is included in SwiftUI and `MFMailComposeViewController` in MessageUI, so it might be redundant. It builds if you remove it